src/options/options.html

```diff
@@ -162,6 +162,20 @@ <h1 data-i18n="__MSG_titleBehaviour__">Behaviour</h1>
 				</ul>
 			</section>
 
+			<!-- Remove "mobile-incompatible" once https://bugzilla.mozilla.org/show_bug.cgi?id=1595822 is fixed -->
+			<section class="mobile-incompatible">
+				<h1 data-i18n="">Context menu</h1>
+				<ul>
+					<li>
+						<div class="line">
+							<input class="setting save-on-change" type="checkbox" id="insertEmoji" data-optiongroup="contextMenu" name="insertEmoji">
+							<label data-i18n="" for="insertEmoji">Insert Emoji</label>
+						</div>
+						<span data-i18n="" class="line indent helper-text">Show context menu item to insert emoji.</span>
```

Comment:
Please fill out the `data-i18n=""` and add the English string as preset in the `messages.json` of the English lang at least. Because this here needs localization.

Reply:
done now

Reply:
Thanks for taking care of this!